app/views/facility_reservations/_reservation.html.haml

```diff
@@ -12,7 +12,7 @@
       %span{ class: order_detail.order_status.to_s.downcase }
         = "- #{order_detail.order_status}"
 
-      - if order_detail.note.present?
+      - if order_detail.note.present? && order_detail.facility.show_order_note?
```

Comment:
seems like this and the logic in calendar_presenter, line 20 are similar. Might want to consider moving this "return order_note if present" logic into the order_detail object itself that can be called from both of these places.

Reply:
I lean towards holding off abstracting this logic until we need it in a 3rd place ([rule of 3](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming))).  These 2 spots are slightly different and the extra abstraction didn't seem worth the work right now.  I also try to avoid adding anything to the OrderDetail class, since it is already 1010 lines long :)